scripts/build-win.py

```diff
@@ -80,21 +80,23 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
     else:
         gn_args.append('rtc_include_tests=false')
         gn_args.append('owt_include_tests=false')
-    if cloud_gaming:
+    if cg_server:
         gn_args.append('rtc_enable_protobuf=false')
         gn_args.append('rtc_enable_win_wgc=false')
-        gn_args.append('owt_cloud_gaming=true')
+        gn_args.append('owt_cg_server=true')
         gn_args.append('enable_libaom=false')
     else:
         gn_args.append('enable_libaom=true')
+    if cg_client:
+        gn_args.append('owt_cg_client=true')
```

Comment:
we are using protobuf on client level, please exclude it from OWT build for cg client otherwise there will be link error with double defined symbols:
```
    if cg_client:
        gn_args.append('owt_cg_client=true')
        gn_args.append('rtc_enable_protobuf=false')
```

Reply:
Done.